src/io.rs

```diff
@@ -134,7 +219,8 @@ pub(crate) fn build_output(
     (stream, config, sender)
 }
 
-pub(crate) fn build_input() -> (Stream, StreamConfig, Receiver<AudioBuffer>) {
+/// Builds the input
+pub fn build_input() -> (Stream, StreamConfig, Receiver<AudioBuffer>) {
```

Comment:
You should port the latency hint to the input builder too (can be done in another PR if it's too much work). They should be symmetrical in the end